examples/puppetmaster/smoketest_e2e.sh

```diff
@@ -4,12 +4,12 @@ set -euo pipefail
 
 # Launches a full Puppet stack and converges a node against it
 
-COMPOSE_PROJECT_NAME=puppetmaster
+COMPOSE_PROJECT_NAME=puppetmaster_$(date +"%Y%m%d_%H%M%S")
```

Comment:
wdyt about something like we use here:
https://github.com/cyberark/secretless-broker/blob/18fbb80b81df905bcb95f229dc90978675891d62/bin/run_integration#L45